tests/test_notifications.py

```diff
@@ -262,7 +261,7 @@ def test_node_subscriptions_and_backrefs_removed_when_node_is_deleted(self):
             subscription.save()
 
             s = getattr(project.creator, 'email_transactional', [])
-            assert_equal(len(s), 1)
+            assert_equal(len(s), 2)
```

Comment:
And:

![screen shot 2016-02-23 at 2 26 43 pm](https://cloud.githubusercontent.com/assets/134455/13263860/8140848c-da39-11e5-8f55-81e8af33f2a3.png) ![screen shot 2016-02-23 at 2 26 34 pm](https://cloud.githubusercontent.com/assets/134455/13263859/81356822-da39-11e5-906a-ab0b55246881.png)